src/cli/MeasureUpdateCommand.hpp

```diff
@@ -57,6 +73,8 @@ namespace cli {
     bool run_tests = false;
 
     unsigned server_port = 0;
+
+    MeasureNewOptions newMeasureOpts;
```

Comment:
This is added to here. because we use make_shared for scoping issues... don't worry about it